src/Admin/Editor/js/activeLineController.js

```diff
@@ -26,11 +29,11 @@ function setActiveLine(cm) {
  * Finds the value of a cookie
  */
 function getCookie(cname) {
-  var name = cname + "=";
-  var decodedCookie = decodeURIComponent(document.cookie);
-  var ca = decodedCookie.split(';');
-  for (var i = 0; i < ca.length; i++) {
-    var c = ca[i];
+  let name = cname + "=";
+  decodedCookie = decodeURIComponent(document.cookie);
+  ca = decodedCookie.split(';');
```

Comment:
You have to use comma here, not semikolon. Otherwise the variable will be registered as global var.
Only last deklaration gets a semikolon at the end
Like so:
```
let  name = cname + "=",
  decodedCookie = decodeURIComponent(document.cookie),
  ca = decodedCookie.split(';');
```

Reply:
done